contrib/src/main/java/gov/nasa/jpl/aerie/contrib/streamline/modeling/Registrar.java

```diff
@@ -66,27 +56,11 @@ public enum ErrorBehavior {
 
   public Registrar(final gov.nasa.jpl.aerie.merlin.framework.Registrar baseRegistrar, final ErrorBehavior errorBehavior) {
     Resources.init();
+    Logging.init(baseRegistrar);
     this.baseRegistrar = baseRegistrar;
     this.errorBehavior = errorBehavior;
-    errors = resource(Discrete.discrete(Map.of()));
-    var errorString = map(errors, errors$ -> errors$.entrySet().stream().map(entry -> formatError(entry.getKey(), entry.getValue())).collect(joining("\n\n")));
-
-    // Register the errors and number of errors resources for output
-    // TODO consider using serializable events, rather than resources, to log errors
-    discrete("errors", errorString, new StringValueMapper());
-    discrete("numberOfErrors", map(errors, Map::size), new IntegerValueMapper());
-  }
-
-  private static String formatError(Throwable e, Collection<String> affectedResources) {
-    return "Error affecting %s:%n%s".formatted(
-        String.join(", ", affectedResources),
-        formatException(e));
-  }
 
-  private static String formatException(Throwable e) {
-    return ExceptionUtils.stream(e)
-        .map(ExceptionUtils::getMessage)
-        .collect(joining("\nCaused by: "));
+    discrete("numberOfErrors", numberOfErrors, new IntegerValueMapper());
```

Comment:
In the event that users have written constraints that depend on the previous resource definition, it may be worth calling out this change in the release notes. Not a reason not to make this change, but worth communicating explicitly.

It also highlights that constraints (and scheduling?) would benefit from being able to refer to simulation events.